WordPress/Classes/ViewRelated/NUX/LoginViewController.m

```diff
@@ -821,10 +632,13 @@ - (void)showCreateAccountView
     [self.navigationController pushViewController:createAccountViewController animated:YES];
 }
 
-- (void)showJetpackAuthentication
+- (void)showJetpackAuthentication:(NSNumber *)blogId
```

Comment:
Good catch. Renamed it here - https://github.com/wordpress-mobile/WordPress-iOS/commit/efed5a89d25944e4fd8803422293c47891b3e2be